### DISCUSSION TITLE
Smiley face appearing under status emoji

### DISCUSSION THREAD

**Discussion by Fdawgs** [community/community#120900] [Mobile] [answered]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.159.0 (129045654)
Device: iPhone 15 Pro (iOS 17.4.1)
</details>

![image](https://github.com/community/community/assets/43814140/f02e091e-acca-40d1-9517-0518fd828813)

Note the smiley face under the umbrella/rain emoji.

**Comment by Iron-Ham** [comment, marked as answer]
Hi Frazer,

Thank you for reporting this issue. The upcoming release of 1.160.0 should have this issue resolved.

Thanks,
Hesham

**Reply by Fdawgs** [reply]
Great, thanks for the update Hesham!

**Reply by Iron-Ham** [reply]
@Fdawgs 1.160 has been released to the App Store today and should be rolling out to all users. Checking in to confirm that the issue as you experienced it is resolved. 

Thanks,
Hesham

**Reply by Fdawgs** [reply]
Thanks @Iron-Ham, I can confirm that the update has resolve this.
